sphinx_gallery/gen_rst.py

```diff
@@ -124,7 +125,7 @@ def flush(self):
 """
 
 
-CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
+CODE_OUTPUT = u""".. rst-class:: sphx-glr-script-out
```

Comment:
(I put in the `__future__` line, removed all `u'` and `u"` and it seems to work fine)